src/contrib/tests/integration-tests.rst

```diff
@@ -10,6 +10,31 @@ The advantage over the unit tests is obvious, the full daemon is ran and you can
 
 The disadvantages also stem from running the tests on the local machine. SSSD relies on whatever server it is connecting to to also run in the test environment provided by the cwrap libraries, but in many cases that is so difficult that we even haven't done the work (e.g. FreeIPA) or outright impossible (Active Directory). Even within the tests themselves, we sometimes stretch the limits of the cwrap libraries. As an example, the socket_wrapper library doesn't support faking the client credentials that the SSSD reads using the ``getsockopt`` call with the ``SO_PEERCRED`` parameter.
 
+Integration tests dependencies
+------------------------------
+Integration tests requires additional packages which are not installed in system by default. One way to solve this is to trigger ``make intgcheck`` in loop and add packages with missing binaries manually. In case of ``Fedora 34`` this will corespond to install following packages:
```

Comment:
s/In case of ``Fedora 34`` this will corespond to install following packages:/To install the dependencies run:

Reply:
Note, the distro is inferred by code-tabs. This way we can easily add other distro if needed.

Reply:
Agree